OpenRA.Mods.Common/Activities/Move/Move.cs

```diff
@@ -388,6 +389,9 @@ abstract class MovePart : Activity
 
 				IsInterruptible = false; // See comments in Move.Cancel()
 
+				TurnsWhileMoving = move.mobile.Info.TurnsWhileMoving;
+				ToFacing = TurnsWhileMoving ? (To - From).Yaw + Move.ActorFacingModifier : toFacing;
```

Comment:
remove the ToFacing set above in line 381

Reply:
It is in line 382, fixed